src/pages/Transfer/index.tsx

```diff
@@ -1,11 +1,75 @@
 
+import * as React from 'react';
 import clsx from 'clsx';
+import { useTranslation } from 'react-i18next';
 
+import useQueryParams from 'utils/hooks/use-query-params';
+import useUpdateQueryParameters, { QueryParameters } from 'utils/hooks/use-update-query-parameters';
+import { QUERY_PARAMETERS } from 'utils/constants/links';
+import Hr1 from 'components/hrs/Hr1';
+import InterlayTabGroup, {
+  InterlayTabList,
+  InterlayTabPanels,
+  InterlayTab,
+  InterlayTabPanel
+} from 'components/UI/InterlayTabGroup';
 import TransferForm from './TransferForm';
+import CrossChainTransferForm from './CrossChainTransferForm';
 import MainContainer from 'parts/MainContainer';
 import Panel from 'components/Panel';
 
+const TAB_IDS = Object.freeze({
```

Comment:
@tomjeatt 
This is not within this scope but could you please move `utils/constants/tab-ids.ts` into `pages/Bridge/index.tsx` as you did in this component?

Reply:
Have a ticket in to tidy up the transfer form so I'll add a note to the ticket and take care of this then.